TestApi/TestApi/Startup.cs

```diff
@@ -103,7 +105,15 @@ public void Configure(IApplicationBuilder app, IWebHostEnvironment env)
         {
             app.RunLatestMigrations();
 
-            if (env.IsDevelopment()) app.UseDeveloperExceptionPage();
+            if (env.IsDevelopment())
+            {
+                app.UseDeveloperExceptionPage();
+            }
+            else if (!SettingsConfiguration.DisableHttpsRedirection)
+            {
+                app.UseHsts();
+                app.UseHttpsRedirection();
+            }
 
             app.UseHttpsRedirection();
```

Comment:
Hi David. Line 168 will ensure it will always UseHttpsRedirection() regardless of the if? Also do you want that to be an else if, or is it better as a separate if statement?